### PR TITLE
How to become member of hubverse@groups.io

### PR DESCRIPTION
**Pull request by mzorn-58**

### CODE REVIEW

**Comment by mzorn-58** [comment]
@nickreich @micokoch easy review needed. Thanks!

File: docs/source/overview/contribute.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
-# How to contribute  
-
-The Consortium of Infectious Disease Modeling Hubs built _the hubverse_ collaboratively and we benefit from your contributions.  
-
-## Use and test our software and tools  
-
-The hubverse was created for teams to easily create and set-up collaborative modeling hubs, and we benefit from having groups use these tools and tell us about it.  
-
-(file_issues)=
```
Comment (nickreich) [inline docs/source/overview/contribute.md:9]: I see this anchoring link is removed. are we sure that it is not linked to elsewhere in the hubdocs?

**Review by nickreich — CHANGES_REQUESTED** [review]
one small comment on this file. Also, it seems like there is a contribute.md file that is in the root directory that should be deleted?

File: docs/source/overview/contribute.md (hunk 1 above)
Comment (mzorn-58) [reply docs/source/overview/contribute.md:9]: @nickreich anchoring link added back in.

**Comment by mzorn-58** [comment]
@nickreich can you merge this?

**Review by nickreich — APPROVED** [review]
i made small changes for markdown and it looks good.

**Pull request merged**